tests/model_selection/test_random_split.py

```diff
@@ -0,0 +1,180 @@
+
+import typing as tp
+from copy import deepcopy
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from rectools import Columns
+from rectools.dataset import Interactions
+from rectools.model_selection import RandomSplitter
+from rectools.model_selection.random_split import get_not_seen_mask
+
+T = tp.TypeVar("T")
+Converter = tp.Callable[[tp.Sequence[int]], tp.List[int]]
+
+
+class TestTimeRangeSplit:
+    @pytest.fixture
+    def shuffle_arr(self) -> np.ndarray:
+        return np.random.choice(np.arange(11), 11, replace=False)
+
+    @pytest.fixture
+    def interactions(self, shuffle_arr: np.ndarray) -> Interactions:
+        df = pd.DataFrame(
+            [
+                [1, 1, 1, "2021-09-01"],
+                [1, 2, 1, "2021-09-02"],
+                [2, 1, 1, "2021-09-02"],
+                [2, 2, 1, "2021-09-03"],
+                [3, 2, 1, "2021-09-03"],
+                [3, 3, 1, "2021-09-03"],
+                [3, 4, 1, "2021-09-04"],
+                [1, 2, 1, "2021-09-04"],
+                [3, 1, 1, "2021-09-05"],
+                [4, 2, 1, "2021-09-05"],
+                [3, 3, 1, "2021-09-06"],
+            ],
+            columns=[Columns.User, Columns.Item, Columns.Weight, Columns.Datetime],
+        ).astype({Columns.Datetime: "datetime64[ns]"})
+        return Interactions(df.iloc[shuffle_arr])
+
+    @pytest.fixture
+    def train_size(self) -> float:
+        return 0.5
+
+    @pytest.mark.parametrize("execution_number", range(5))
+    def test_without_filtering(self, interactions: Interactions, train_size: float, execution_number: list) -> None:
+        interactions_copy = deepcopy(interactions)
+        rs = RandomSplitter(train_size, False, False, False)
+
+        actual = list(rs.split(interactions, collect_fold_stats=True))
+        pd.testing.assert_frame_equal(interactions.df, interactions_copy.df)
+        assert len(actual) == 1
+        assert len(actual[0]) == 3
+
+        assert actual[0][0].shape[0] == int(train_size * interactions.df.shape[0])
+        assert actual[0][0].shape[0] + actual[0][1].shape[0] == interactions.df.shape[0]
+
+        fold_info = actual[0][2]
+        df = interactions.df
+        train_users = df.iloc[actual[0][0]]["user_id"]
+        train_items = df.iloc[actual[0][0]]["item_id"]
+        test_users = df.iloc[actual[0][1]]["user_id"]
+        test_items = df.iloc[actual[0][1]]["item_id"]
+
+        assert fold_info["Train"] == train_users.size
+        assert fold_info["Train users"] == pd.unique(train_users).size
+        assert fold_info["Train items"] == pd.unique(train_items).size
+        assert fold_info["Test"] == test_users.size
+        assert fold_info["Test users"] == pd.unique(test_users).size
+        assert fold_info["Test items"] == pd.unique(test_items).size
+
+    @pytest.mark.parametrize("execution_number", range(5))
+    def test_cold_users(self, interactions: Interactions, train_size: float, execution_number: list) -> None:
```

Comment:
Can you please name tests the same as tests for `TimeRangeSplit` (for consistency)